### PR TITLE
{bio}[GCC/11.3.0] Satsuma2 v20220304

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by anselmicz** [comment]
Test report by @anselmicz
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login4.karolina.it4i.cz - Linux CentOS Linux 7 (Core), x86_64, AMD EPYC 7452 32-Core Processor, Python 3.10.4
See https://gist.github.com/4a1a30a7ce422f09f5223aaa378bbcc1 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/21509fc2f8be51344dba2521a5d33153 for a full test report.

File: easybuild/easyconfigs/s/Satsuma2/Satsuma2-2022-03-04-GCC-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'Satsuma2'
+local_commit = '37c5f38'
+version = '2022-03-04'
+
+homepage = 'https://github.com/bioinfologics/satsuma2'
+description = """Satsuma2 is an optimised version of Satsuma, a tool to reliably align large and complex DNA sequences
+providing maximum sensitivity (to find all there is to find), specificity (to only find real homology) and speed (to
+accomodate the billions of base pairs in vertebrate genomes)."""
```
Comment (verdurin) [inline easybuild/easyconfigs/s/Satsuma2/Satsuma2-2022-03-04-GCC-11.3.0.eb:10]:
```suggestion
accommodate the billions of base pairs in vertebrate genomes)."""
```

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/s/Satsuma2/Satsuma2-2022-03-04-GCC-11.3.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/s/Satsuma2/Satsuma2-2022-03-04-GCC-11.3.0.eb]: Nice catch. This was copy-pasted from https://github.com/bioinfologics/satsuma2/blob/master/README.md, so maybe send them a PR too? ;)

File: easybuild/easyconfigs/s/Satsuma2/Satsuma2-2022-03-04-GCC-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'Satsuma2'
+local_commit = '37c5f38'
+version = '2022-03-04'
```
Comment (boegel) [inline easybuild/easyconfigs/s/Satsuma2/Satsuma2-2022-03-04-GCC-11.3.0.eb:5]: Hmm, using `20220304` would fit better with other easyconfigs where we use a datestamp as a workaround for the lack of a proper version, so I'll change that...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3100.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/fbe1af24d6342faa06d798790d221164 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/0a026bd894b434fc856c0cf7326b2a85 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**